provider-ui/.eslintrc

```diff
@@ -0,0 +1,62 @@
+module.exports = {
```

Comment:
This change is necessary for MUI v5 compatibility?

Reply:
Not directly, sorry this was more to Next.js 13